workers/unity/Assets/Generated/Source/improbable/gdk/tests/blittabletypes/BlittableComponentReaderWriter.cs

```diff
@@ -16,41 +16,72 @@ public partial class BlittableComponent
         [ComponentId(1001)]
         internal class ReaderWriterCreator : IReaderWriterCreator
         {
-            public IReaderInternal CreateReaderWriter(Entity entity, EntityManager entityManager, ILogDispatcher logDispatcher)
+            public IReaderWriterInternal CreateReaderWriter(Entity entity, EntityManager entityManager, ILogDispatcher logDispatcher)
             {
                 return new ReaderWriterImpl(entity, entityManager, logDispatcher);
             }
         }
 
         [ReaderInterface]
         [ComponentId(1001)]
-        public interface Reader
+        public interface Reader : IReader<SpatialOSBlittableComponent, SpatialOSBlittableComponent.Update>
         {
         }
 
-        public class ReaderWriterImpl : Reader, IReaderInternal
+        [WriterInterface]
+        [ComponentId(1001)]
+        public interface Writer 
+        {
+        }
+
+        internal class ReaderWriterImpl : IReaderWriterInternal, Reader, Writer
```

Comment:
No, because the SpatialOSBehaviourLibrary (which btw I really want to rename, also in above ticket) looks at the types [Require]d by the user and needs to tell the component ID for those. And that's the reader/writer interface types.